spec/models/calculator_spec.js

```diff
@@ -600,4 +601,19 @@ describe('Calculator', () => {
       expect(result['statement_results']['DiabetesFootExam']['SDE Sex']['pretty']).toEqual('[PatientCharacteristicSex\nCODE: AdministrativeGender M]');
     });
   });
+
+  describe('namingsystem', () => {
+    it('is not null', () => {
+      const valueSetsByOid = getJSONFixture('measures/CMS108v7/value_sets.json');
+      const measure = getJSONFixture('measures/CMS108v7/CMS108v7.json');
+      const patients = [getJSONFixture('patients/CMS108v7/INR4no_decimal_DayOfAnes_NUMERPass.json')];
+      const options = { doPretty: true };
+      const calculationResults = Calculator.calculate(measure, patients, valueSetsByOid, options);
+      const result = Object.values(calculationResults[Object.keys(calculationResults)[0]])[0];
+
+      const sampleResult = result['statement_results']['VenousThromboembolismProphylaxis']['Is In Low Risk for VTE or On Anticoagulant'];
+      const sampleResultId = sampleResult.raw[0].id.value;
+      expect(sampleResult['pretty']).toEqual(`[{\n  id: {\n    value: "${sampleResultId}",\n    namingSystem: null\n  },\n  authorDatetime: 11/01/2012 10:00 AM\n}]`);
```

Comment:
Test description is namingsystem is not null, yet the test expects the namingSystem to be null